### PR TITLE
Update build.yml for publishing docker images

### PR DESCRIPTION
**Pull request by kiliczsh**
- [ ] Add repository secrets: DOCKER_HUB_USERNAME, DOCKER_HUB_ACCESS_TOKEN
- [x] Update muhammeddkilicc/podiscover tag

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -1,18 +1,29 @@
...
-      - name: Build Docker Image
-        run: docker build -t podiscover:${{github.sha}} .
+      - name: Set up Docker
+        uses: docker/setup-buildx-action@v1
+
+      - name: Log in to Docker Hub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKER_HUB_USERNAME }}
+          password: ${{ secrets.DOCKER_HUB_ACCESS_TOKEN }}
```
Comment (enderahmetyurt) [inline .github/workflows/build.yml:22]: do we need these env or somewhere?

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -1,18 +1,29 @@
...
+
+      - name: Build and push Docker image
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          push: true
+          tags: muhammeddkilicc/podiscover:${{ github.sha }}
```
Comment (enderahmetyurt) [inline .github/workflows/build.yml:29]: could we change name of tags?

**Review by enderahmetyurt — APPROVED** [review]

**Review by enderahmetyurt — COMMENTED** [review]
nice touch. I asked some questions.

File: .github/workflows/build.yml (hunk 1 above)
Comment (enderahmetyurt) [inline .github/workflows/build.yml:22]: do we need to add them to env or where something similar?

Comment (kiliczsh) [reply .github/workflows/build.yml:22]: It should be added to Secrets for Actions, here how I did: 

Screenshot 2023-04-20 at 09 06 41

File: .github/workflows/build.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (kiliczsh) [reply .github/workflows/build.yml:29]: Yes, this is how I tested. If you give me docker hub username I can replace with muhammeddkilicc

[the remark this answers is not in the document]
Comment (enderahmetyurt) [reply .github/workflows/build.yml:29]: `eaydev` thank you 🙏

[the remark this answers is not in the document]
Comment (kiliczsh) [reply .github/workflows/build.yml]: done

**Review by enderahmetyurt — APPROVED** [review]
I added `DOCKER_HUB_USERNAME, DOCKER_HUB_ACCESS_TOKEN` to Secrets for Actions in Podiscover. I think you would merge it now.

**Comment by kiliczsh** [comment]
This is the [docker image](https://hub.docker.com/r/eaydev/podiscover) produced. 🚀

**Pull request merged**